keys.go

```diff
@@ -81,6 +81,7 @@ type Key struct {
 	PurgeAllowedFrom    *time.Time  `json:"purgeAllowedFrom,omitempty"`
 	PurgeScheduledOn    *time.Time  `json:"purgeScheduledOn,omitempty"`
 	DualAuthDelete      *DualAuth   `json:"dualAuthDelete,omitempty"`
+	Sort                string      `json:"sort,omitempty"`
```

Comment:
I don't think this is needed or correct?

Its not used in the example tests either.